release-notes/aws/v11.0.2.md

```diff
@@ -0,0 +1,28 @@
+## :zap: Giant Swarm Release 11.0.2 for AWS is now active for you! :zap:
+
+This release includes some reliability and user experience improvements.
+
+Below, you can find more details on components that were changed with this release.
+
+### cert-manager v0.9.0 ([Giant Swarm app v1.0.6](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#v106-2020-02-28))
+
+- Configured app icon.
+
+### chart-operator [v0.12.0](https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0120-2020-03-09)
```

Comment:
@cokiengchiara @sslavic @tomahawk28 Added this to each affected releases. PTAL

Reply:
I would have this changelog for https://github.com/giantswarm/releases/pull/149 too. Thanks.